web/src/components/common/UI/Table.tsx

```diff
@@ -287,6 +287,32 @@ const Table: React.FC<ITableProps> = (props) => {
     [],
   );
 
+  const findRowByRoleInfo = (rolwTosearch: object[], searchKey: string) => {
+    return rolwTosearch.find(
+      (a: any) =>
```

Comment:
variable name and type haven't been addressed here

Reply:
sorry for being careless, should be good this time.